docs/manual/developer_guide.adoc

```diff
@@ -1488,6 +1488,12 @@ audit_file_contents::
 ** *contents* - expected contents of the file
 * Languages: Ansible, Bash, OVAL
 
+audit_rules_time::
+* Ensure there is an audit rule that records time-related syscalls. The syscall would be logged using the `audit_time_rules` key
+* Parameters:
+** *syscall* - name of the system call, e.g. `settimeofday`
+* Languages: Kubernetes
```

Comment:
Could the template be more generic (`audit_syscall_rule`) and the key parametrized?
There are a lot of audit rules about syscall that could benefit.